pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -62,6 +63,8 @@ var (
 	storageClassName         = "test-storage-class"
 )
 
+const defaultLocalDNS = "svc.cluster.local"
```

Comment:
Could you please use the constant defined in the dns package, dns.DefaultLocalDNS, instead of re creating this? This comment pertains to the other tests as well.